ChangeLog

```diff
@@ -12,6 +12,8 @@ What's New in astroid 2.10.1?
 =============================
 Release date: TBA
 
+* Add new (optional) ``doc_node`` attribute to ``nodes.Module``, ``nodes.ClassDef``,
```

Comment:
Ah, that was one remaining question. Technically this should be in `2.11` I think as this can't really be considered a bug fix.
Do we want to release `2.11` with only this? Or do we keep it in `2.10.1`?

Reply:
Though about this too. Let's move it to `2.11`. I've renamed to existing project milestones.
`2.10.1` -> `2.11.0` and `2.11.0` -> `2.12.0`.